urbanopt/geojson-modelica-translator
# update license run script

#### Any background context you want to provide?

The update license script was adding the copyright to jmodelica and shouldn't be.

#### What does this PR accomplish?
* fix script
* rename contest to conftest.py

#### How should this be manually tested?
* Run `./setup.py update_licenses`
* Verify that the diff doesn't chnage

#### What are the relevant tickets?
N/A
#### Screenshots (if appropriate)
